ui/apps/ui/src/app/layouts/content-box/content-box.component.ts

```diff
@@ -1,66 +1,66 @@
-import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
-import truncate from 'lodash-es/truncate';
-import { attachHighlightsToTxt, stripHighlightedFromHtml } from '../utils';
-import { stripHtml } from 'string-strip-html';
+// import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
```

Comment:
let's not keep the commented code in the codebase - we can always return to it by just using git

Reply:
I've commented it to check if it's really not used and forgot to delete it ;)